### PR TITLE
Added pause command

### PR DESCRIPTION
**Pull request by bobbycNH**
Pretty self explanatory, adds support for the 'PlayPause' method.

### CODE REVIEW

File: helm-rhythmbox.el [hunk 1]
```diff
@@ -120,6 +120,15 @@ formatted with `helm-rhythmbox-candidate-format'."
       (dbus-call-method :session service path interface
                         "AddToQueue" (rhythmbox-song-uri song)))))
 
+(defun helm-rhythmbox-pause-song ()
+  (interactive)
+  "Play/pause the selected song."
```
Comment (mrBliss) [inline helm-rhythmbox.el:124]: The docstring should come before `(interactive)`.

File: helm-rhythmbox.el [hunk 2]
```diff
@@ -120,6 +120,15 @@ formatted with `helm-rhythmbox-candidate-format'."
       (dbus-call-method :session service path interface
                         "AddToQueue" (rhythmbox-song-uri song)))))
 
+(defun helm-rhythmbox-pause-song ()
```
Comment (mrBliss) [inline helm-rhythmbox.el]: Maybe incorporate `play` in the title, something like `helm-rhythmbox-play/pause-song`?

Comment (bobbycNH) [inline helm-rhythmbox.el]: I tend to read slashes as module separators, so `helm-rhythmbox-play/pause-song` reads funny to me.  Maybe just `helm-rhythmbox-playpause-song`?

Comment (bobbycNH) [inline helm-rhythmbox.el]: I should mention that I'm not much of an elisp hacker, so stylistically your suggestion may be more appropriate.

Comment (mrBliss) [inline helm-rhythmbox.el]: `helm-rhythmbox-playpause-song` is ok for me.

**Pull request merged**